HealthcareSystem/UserServiceTests/IntegrationTests/UserServiceTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Backend.Model;
+using Microsoft.AspNetCore.Mvc;
+using UserService.CustomException;
+using UserService.Model;
+using UserService.Repository;
+using UserService.Repository.Implementation;
+using UserService.Service.Implementation;
+using Xunit;
+
+namespace UserServiceTests.IntegrationTests
+{
+    public class UserServiceTests
+    {
+        private UserService.Service.Implementation.UserService SetupRepositoryAndService()
+        {
+            DbContextInMemory testData = new DbContextInMemory();
+            testData.SeedDatas();
+            MyDbContext context = testData._context;
+            var userRepo = new UserRepository(context);
+            return new UserService.Service.Implementation.UserService(userRepo);
+        }
+
+        [Fact]
+        public void SuccessLoginPatient()
+        {
+            UserService.Service.Implementation.UserService userService = SetupRepositoryAndService();
+            UserAccount userAccount = userService.GetByEmailAndPassword("ana_anic98@gmail.com", "11111111");
+
+            Assert.False(userAccount is null);
+        }
+
+        [Fact]
+        public void SuccessLoginAdmin()
+        {
+            UserService.Service.Implementation.UserService userService = SetupRepositoryAndService();
+            UserAccount userAccount = userService.GetByEmailAndPassword("milic_milan@gmail.com", "milanmilic965");
+
+            Assert.False(userAccount is null);
```

Comment:
I think you should maybe Assert that the correct data is returned here, not just that it's not null.